proc/process.c

```diff
@@ -361,18 +364,39 @@ static void process_illegal(unsigned int n, exc_context_t *ctx)
 }
 
 
-#ifndef NOMMU
+static void assign_tls(hal_tls_t *process_tls, hal_tls_t *tls, void *tbss_addr)
```

Comment:
Use module prefix, on what -> do what nomenclature and camelCase:
`process_tlsAssign`